README.md

```diff
@@ -1,8 +1,13 @@
 # MLab Visualizations
 
-[![Build Status](https://travis-ci.org/bocoup/mlab-vis-client.svg?branch=master)](https://travis-ci.org/bocoup/mlab-vis-client)
+[![Build Status](https://travis-ci.org/opentechinstitute/mlab-vis-client.svg?branch=mlab-staging)](https://travis-ci.org/opentechinstitute/mlab-vis-client)
 
```

Comment:
so we don't actually need this change if we want to merge this to production. I changed this on the staging branch so that the "status" would reflect status of the right build.